root/etc/s6-overlay/s6-rc.d/init-config-immich/run

```diff
@@ -4,10 +4,10 @@
 # make folders
 mkdir -p \
     /config/machine-learning/models \
-    "$IMMICH_MEDIA_LOCATION"
+    "${IMMICH_MEDIA_LOCATION}"
 
 # permissions
-find "$IMMICH_MEDIA_LOCATION" -maxdepth 0 \( ! -user abc -o ! -group abc \) -exec lsiown -R abc:abc {} \;
 find /app/immich -path "*/node_modules" -prune -o -exec chown abc:abc {} +
 lsiown -R abc:abc \
     /config
+[[ "${APPLY_PERMISSIONS}" == "true" ]] && lsiown -R abc:abc "${IMMICH_MEDIA_LOCATION}"
```

Comment:
I'm fine with it but maybe we should have it for `/config` too ?

Reply:
> 

/config should always be mounted locally, and not our problem if mounted any other way IMO